### DISCUSSION TITLE
Error: Command failed: logrotate /etc/logrotate.d/nginx-proxy-manager

### DISCUSSION THREAD

**Discussion by masterwishx** [NginxProxyManager/nginx-proxy-manager#3400] [Q&A]
using latest version ,i have permissions issue in ubuntu with root user:

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/28630321/4c220fa5-38ee-4415-ac94-8b67850badf0)
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/28630321/d0174139-a712-4dd7-a273-739a29c58872)

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/28630321/f080e457-816a-41e3-bf43-a60e78370266)

**Comment by masterwishx** [comment]
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/28630321/ee48d37a-28c0-4e04-9417-e41be9494ad9)

**Comment by FibreTTP** [comment]
The `/data` directory in your instance has full permissions. It's not supposed to be like that. You've probably used `chmod` on it from outside the Docker container.

logrotate is complaining because `/data/logs` also has full permissions. You will need to run the command:
`docker exec <npm-container-id> chmod -R a=rX,u+w /data/logs`
to fix this.

My PR that you commented on (#3219) doesn't have to do with this (it fixes permissions for when the user is **not** running as root, but still works when they are). The issue you commented on (#2938) is different from your issue (they're not running as root).

**Reply by masterwishx** [reply]
So you running as 1000:1000?

**Reply by FibreTTP** [reply]
I've created a management user and group and run all my management services that support changing users as such user and group. It's optional, but theoretically more secure. Here's how you'd do it on a clean install: https://nginxproxymanager.com/advanced-config/#running-processes-as-a-user-group

**Reply by masterwishx** [reply]
OK ,thanks a lot for help.
 runned docker exec `nginx-proxy-manager chmod -R 644 /data/logs` and all fine now
